.circleci/config.yml

```diff
@@ -685,6 +685,7 @@ jobs:
         command: |
           if [ -f ~/workbench/jira-env.sh ]; then
             source ~/workbench/jira-env.sh
+            echo "export JIRA_ISSUE_KEY=\"JIRA_ISSUE_KEY\"" >> $BASH_ENV
```

Comment:
this is generally followed by `source $BASH_ENV` and I suspect that step is necessary

Reply:
Good catch. Thanks!